### ISSUE TITLE
graph.detach().filter().subgraph() on GraphView converts dtype of 't' column from i32 to i64

### ISSUE THREAD

**Issue by TeunHuijben** [royerlab/tracksdata#248]
In the following snippet the dtype of the time column between `graph` and `new_graph` is different (`i32` vs `i64`). Is this intended or a bug?

```
    kwargs = {
        "drivername": "sqlite",
        "database": ":memory:",
        "overwrite": True,
    }
    graph = td.graph.SQLGraph(**kwargs)
    graph = graph.filter().subgraph()
    new_graph = graph.detach().filter().subgraph()
```

**Comment by TeunHuijben** [comment]
Solved!
